src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
+        int value = new Random().nextInt(6);
```

Comment:
create a global Random field and use it here, so you won't need to create its instance each time when you call this method